Sources/TuistLoaderTesting/Loaders/TestData/ProjectDescription+TestData.swift

```diff
@@ -142,7 +142,7 @@ extension TestAction {
     public static func test(targets: [TestableTarget] = [],
                             arguments: Arguments? = nil,
                             configuration: ConfigurationName = .debug,
-                            coverage: Bool = true) -> TestAction
+                            coverage _: Bool = true) -> TestAction
```

Comment:
This should be passed to `TestAction.targets` below

Reply:
ah good spot we missed this!

https://github.com/tuist/tuist/pull/3520